src/framework/preferences/js/SyllabificationEnactor.js

```diff
@@ -37,14 +37,16 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
             }
         },
         selectors: {
-            separator: ".flc-syllabification-separator"
+            separator: ".flc-syllabification-separator",
+            softHyphenPlaceholder: ".flc-syllabification-softHyphenPlaceholder"
         },
         strings: {
             languageUnavailable: "Syllabification not available for %lang",
             patternLoadError: "The pattern file %src could not be loaded. %errorMsg"
         },
         markup: {
-            separator: "<span class=\"flc-syllabification-separator fl-syllabification-separator\"></span>"
+            separator: "<span class=\"flc-syllabification-separator fl-syllabification-separator\"></span>",
+            softHyphePlaceholder: "<span class=\"flc-syllabification-softHyphenPlaceholder\"></span>"
```

Comment:
Typo name softHyphePlaceholder which at least was copy and pasted correctly at line 161 : P